plugins/scaffolder-node/src/actions/types.ts

```diff
@@ -105,3 +105,9 @@ export type TemplateAction<
   };
   handler: (ctx: ActionContext<TActionInput, TActionOutput>) => Promise<void>;
 };
+
+export type JsonType = {
+  [key: string]: string;
```

Comment:
this is too strict, as the type of values can also be something else

Reply:
when dealing with values coming from a `catalog-info.yaml` we end up fixing the value as a string, but really depending on the case it can be a number or a boolean, I think it would be interesting to have a maximum of these 3 types. Putting an "any" type leaves the possibilities wide open. 🤔